samples/NugetDeployment/windowsservercore-docker-entrypoint.ps1

```diff
@@ -0,0 +1,5 @@
+# Workaround from # Workaround from https://github.com/docker/for-win/issues/1976#issuecomment-585423014
```

Comment:
```suggestion
# Workaround from https://github.com/docker/for-win/issues/1976#issuecomment-585423014
```

Reply:
Fixed in a6303c60daa4f2ed97dc5e6d2a455e56dda9fea4